native/src/__tests__/NavigatorContainer.spec.tsx

```diff
@@ -28,15 +30,14 @@ jest.mock('../constants/NativeConstants')
 jest.mock('react-native-share')
 const mockStore = configureMockStore()
 
-class MockNavigator extends React.Component<void> {
-  render() {
-    return null
-  }
+const MockNavigator: React.FC = () => {
+  return null
```

Comment:
Little bit hacky, but basically `React.FC` can also return null elements and is a ReactElement.

Reply:
I'm fine with that :)